song-server/src/main/java/bio/overture/song/server/controller/AnalysisController.java

```diff
@@ -82,7 +81,7 @@
       @ApiParam(value = "Non-empty comma separated list of analysis states to filter by")
           @RequestParam(value = "analysisStates", defaultValue = "PUBLISHED", required = false)
           String analysisStates) {
-    return analysisService.getAnalysis(studyId, ImmutableSet.copyOf(COMMA.split(analysisStates)));
+    return analysisService.getAnalysis(studyId, ImmutableSet.copyOf(",".split(analysisStates)));
```

Comment:
Create a `Splitters` class with a no args constructor, that contains:
`public static final Splitter COMMA = Splitter.on(",");`
and likewise for any other splits.